server/src/main/java/org/elasticsearch/action/support/IndicesOptions.java

```diff
@@ -103,6 +105,10 @@ private static void updateSetForValue(EnumSet<WildcardStates> states, String wil
         public static final EnumSet<Option> NONE = EnumSet.noneOf(Option.class);
     }
 
+    private static final DeprecationLogger DEPRECATION_LOGGER = DeprecationLogger.getLogger(IndicesOptions.class);
+    private static final String IGNORE_THROTTLED_DEPRECATION_MESSAGE = "[ignore_throttled] parameter is deprecated, " +
+        "because frozen indices have been deprecated. Migrate to use frozen tier instead and stop using [ignore_throttled] parameter.";
```

Comment:
Nit: comma is not necessary. I also wonder if either cold or frozen tiers could be recommended in place of frozen indices depending on the use case?
```suggestion
    private static final String IGNORE_THROTTLED_DEPRECATION_MESSAGE = "[ignore_throttled] parameter is deprecated " +
        "because frozen indices have been deprecated. Consider cold or frozen tiers in place of frozen indices.";
```